src/screens/EWayBill/component/AddTransporterModalize.tsx

```
@@ -0,0 +1,128 @@
+                        lable="Transporter ID"
+                        isRequired={true}
+                        containerStyle={styles.inputFieldStyle}
+                        placeholderTextColor={theme.colors.secondary}
+                        value={transporterId}
+                        onChangeText={(text) => {
+                            setTransporterId(text);
+                        }}
+                    />
+                    <View style={styles.modalView}>
+                        <TouchableOpacity
+                            style={[styles.createButton,{backgroundColor: (false) ? '#E6E6E6' :'#5773FF'}]}
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)